event_manager/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.contrib.auth.models import User
+
+class member(models.Model):
+    user=models.ForeignKey(User,on_delete=models.CASCADE)
+    last_name=models.CharField(max_length=200)
+    netid=models.IntegerField(default=0)
+    role=models.CharField(max_length=200)
+    major=models.CharField(max_length=200)
+    points=models.IntegerField(default=0)
+
+class teams(models.Model):
+    user=models.ForeignKey(member,on_delete=models.CASCADE)
```

Comment:
I think the foreign key is going to refer one team to one member, which is not the functionality we want.

Reply:
we can do one to many with this as well, https://stackoverflow.com/questions/6928692/how-to-express-a-one-to-many-relationship-in-django

Reply:
Correct. However, this is creating a one-user-to-many-teams relationship. Can we move the foreign key to the member class?

Reply:
can one use be part of multiple teams? i think we should do many to many then, otherwise we can move foreign key to member model

Reply:
That's a good idea. I think Shakwan said that many members "share" teams. 👍